typesystem/json_schema.py

```diff
@@ -435,7 +435,9 @@ def to_json_schema(
         return data
 
     field_type = type(field)
-    raise TypeError("Unsupported field type {field_type} passed to 'to_json_schema'")
+    raise TypeError(
+        f"Unsupported field type {field_type.__qualname__!r} passed to 'to_json_schema'"
```

Comment:
Great - yup. I'd prefer if we can keep attribute lookups pulled outside of `f-strings`, so perhaps...

```python
field_qualname = type(field).__qualname__
raise TypeError("Unsupported field type {field_qualname!e} passed to 'to_json_schema'")
```

Or something similar?

Reply:
simplified it even more